test/functional/cmdLineTests/valuetypeddrtests/flattened32bitRefBackfillTest.xml

```diff
@@ -90,9 +90,9 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 			<input>!j9object $flatSingleBackfillInstance$</input>
 			<input>quit</input>
 		</command>
-		<output regex="no" type="success" showMatch="yes">l (offset = 4) (ValueLong)</output>
-		<output regex="no" type="success" showMatch="yes">o (offset = 12) (ValueObject)</output>
-		<output regex="no" type="success" showMatch="yes">i (offset = 0) (ValueInt)</output>
+		<output regex="yes" javaUtilPattern="yes" type="success" showMatch="yes">l \(offset = 4\) \(.*ValueLong\)</output>
```

Comment:
This is success condition match, instead of using regex `.*ValueLong`, `.*ValueObject`, etc, better to use full class name.

Reply:
The meaning of `.*` is nothing for qtypes and `org/openj9/test/lworld/ValueTypeTestClasses$` for the lw5 tests.

I'm trying out having exact matches but the `$` is causing issues with the cmdline test framework since `$` is used to indicate the start of a variable. What do you think about something like this with regex to make the matches more specific?

Replace `.*` with `(|org\/openj9\/test\/lworld\/ValueTypeTestClasses.)`? Instead of matching any characters it expects either nothing or org/openj9/test/lworld/ValueTypeTestClasses followed by any single character to deal with the $.

Reply:
> The meaning of .* is nothing for qtypes and org/openj9/test/lworld/ValueTypeTestClasses$ for the lw5 tests.

Can you use 2 non-regex success conditions for this ? e.g.
```
<output regex="no" type="success" showMatch="yes">l (offset = 4) (ValueLong)</output>
<output regex="no" type="success" showMatch="yes">l (offset = 4) (org/openj9/test/lworld/ValueTypeTestClasses$ValueLong)</output>
```

Reply:
The tests will fail at https://github.com/eclipse-openj9/openj9/blob/master/test/functional/cmdline_options_tester/src/TestSuite.java#L289 if a `$` character is used because it expects there to be a closing `$` to define a variable.

Such as https://github.com/eclipse-openj9/openj9/blob/master/test/functional/cmdLineTests/valuetypeddrtests/unflattenedvaluetypeddrtests.xml#L55

Reply:
OK, then I think we can go with the current change. Thanks for trying things out.